build/debian-hyperkube-base/Makefile

```diff
@@ -23,7 +23,7 @@ TAG=0.10
 ARCH?=amd64
 CACHEBUST?=1
 
-BASEIMAGE=k8s.gcr.io/debian-base-$(ARCH):0.3
+BASEIMAGE=k8s.gcr.io/debian-base-$(ARCH):0.3.1
```

Comment:
should this also update the tag for debian-hyperkube-base?

Reply:
good catch, bumped to 0.11